docs/de/guide/nuxt.md

```diff
@@ -30,7 +30,7 @@ yarn add three @tresjs/nuxt
 
 - 🤓 Automatischer Import von Komponenten und Composables aus dem [TresJS-Ökosystem](https://github.com/orgs/Tresjs/repositories)
 - `TresCanvas` ist nur auf dem Client verfügbar, daher ist es nicht notwendig, `.client` zum Namen der Komponente hinzuzufügen oder `<ClientOnly />` zu verwenden
-- Konfiguriert automatisch den Vue-Compiler, um TresJS-Komponenten zu unterstützen, siehe [warum](http://localhost:5173/de/guide/troubleshooting.html#fehler-beim-auflosen-des-komponenten-trescomponent)?
```

Comment:
@Tinoooo I fixed this link in the english, here and in the spanish version, where for some reason this anchor was created, after shortening it and removing the extra parts the links still worked as expected